Kernel/API/KeyCode.h

```diff
@@ -167,3 +172,13 @@ inline const char* key_code_to_string(KeyCode key)
         return nullptr;
     }
 }
+
+inline KeyCode string_to_key_code(const String& str)
+{
+    int n = sizeof(str_key_codes) / sizeof(str_key_codes[0]);
+    for (int i = 0; i < n; i++) {
+        if (String(str_key_codes[i]) == String(str.characters()))
```

Comment:
You can use `StringView` instead, which will use the existing string memory in `str_key_codes`.
```suggestion
        if (StringView { str_key_codes[i] } == str)
```

Reply:
For some reason including strcmp was causing redefinition errors. This will be resolved.